src/include/btree.h

```diff
@@ -93,6 +93,23 @@ typedef enum { /* Start position for eviction walk */
 /* An invalid btree file ID value. ID 0 is reserved for the metadata file. */
 #define WT_BTREE_ID_INVALID UINT32_MAX
 
+/*
+ * WT_ADDR_COPY --
+ *	We have to lock the WT_REF to look at a WT_ADDR: a structure we can use to quickly get a
+ * copy of the WT_REF address information.
+ */
+struct __wt_addr_copy {
```

Comment:
Why did this get moved?

Reply:
What about putting this struct is `bt_mem.h` instead? Below where `__wt_addr` is defined?

Reply:
My first change was in the `bt_mem.h` file only, but it has a dependency with a macro from the `btree.h`. Due to that reason, I moved it into the `btree.h` file.

Reply:
I see, perhaps it's nit pick and extra work but wanted to point it out any way - 
It seems reasonable renaming this macro to `WT_ADDR_MAX_COOKIE`; and moving it along with `__wt_addr_copy` to `btmem.h` - for the sake of similar structs being in the same file. 

I leave that up to you tho, so feel free to resolve this one :-)

Reply:
Yes, it is a nice idea. I moved the macro and also the structure into the btmem.h file.